### PR TITLE
Updating using two editors docs

### PR DESCRIPTION
**Pull request by filipsobol**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Update "Using two editors" docs.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Review by godai78 — APPROVED** [review]

File: docs/installation/advanced/using-two-editors.md [hunk 1]
```diff
@@ -29,91 +29,88 @@ You can start by forking (or copying) an existing build like in the {@link insta
...
 Now it is time to add the missing editor package and install it:
 
-```
-npm install --save-dev @ckeditor/ckeditor5-editor-inline
+```bash
+yarn add -D @ckeditor/ckeditor5-editor-inline
 ```
 
-Once all the dependencies are installed, change the webpack's entry point which is the `src/ckeditor.js` file. For now, it was exporting just a single class:
+Once all the dependencies are installed, you will need to modify the `src/ckeditor.ts` file, which currently only exports a single class. The first step is to move all plugins and configuration to variables so they can be reused by both editors:
```
Comment (gorzelinski) [inline docs/installation/advanced/using-two-editors.md:41]:
```suggestion
After installing all the dependencies, modify the `src/ckeditor.ts` file, which currently only exports a single class. The first step is to move all plugins and configuration to variables so they can be reused by both editors:
```

Maybe we could avoid passive voice?

File: docs/installation/advanced/using-two-editors.md [hunk 2]
```diff
@@ -29,91 +29,88 @@ You can start by forking (or copying) an existing build like in the {@link insta
...
+ClassicEditor.builtinPlugins = plugins;
+ClassicEditor.defaultConfig = config;
+
+export {
+  ClassicEditor
 };
 ```
 
-Let's make it export an object with two classes: `ClassicEditor` and `InlineEditor`. To make both constructors work in the same way (load the same plugins and default configuration) you also need to assign `builtinPlugins` and `defaultConfig` static properties to both of them:
+Now you can add the `InlineEditor` class to the file, add the same plugins and configuration to it and export it:
```
Comment (gorzelinski) [inline docs/installation/advanced/using-two-editors.md:66]:
```suggestion
Now, you can add the `InlineEditor` class to the file. Add the same plugins and configuration to the class and then export it:
```

I would change the punctuation a little.

**Review by gorzelinski — COMMENTED** [review]
I added some suggestions, but merging without them would be ok.

**Comment by filipsobol** [comment]
I like these suggestions. I will make another PR @gorzelinski

**Pull request merged**